audio/mpd/Portfile

```diff
@@ -147,6 +136,14 @@ configure.args \
 
 configure.cflags-append -I${prefix}/include
 
+# Wrap the macOS12 functions, may be added into legacy-support see PR49
+if {${os.major} < 21} {
```

Comment:
This could be applied flat since the wraps only get applied for the affected systems.